packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,105 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS emailTypes (
+  id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  -- these are intended to be email template names
+  emailType VARCHAR(255) NOT NULL,
+  UNIQUE emailTypeIndex (emailType)
+) ENGINE=InnoDB;
+
+INSERT INTO emailTypes (emailType) VALUES
```

Comment:
Would adding a new email template require a database migration? We don't add template often but I can see this as a big barrier.